### ISSUE TITLE
Tile source collision lines not showing up

### ISSUE THREAD

**Issue by OneWhoBuilds** [defold/defold#9357]
_Is it a private platform, and a platform specific issue? (i.e. would it break your NDA?)<br>
Then please report your issue in the corresponding extension repository!_

**Describe the bug (REQUIRED)**
The coloured lines that show which tile is in which collision group are not appearing

**To Reproduce (REQUIRED)**
1. Go to a tile_set.tilesource
2. See error

**Expected behavior (REQUIRED)**
I expect the lines to appear

**Defold version (REQUIRED):**
  - Version 1.9.2

**Platforms (REQUIRED):**
 - Platforms: Windows
 - OS: windows 11 (Dev preview - 24H2 Build 26120.1542 )
 - Device: PC

**Minimal repro case project (OPTIONAL):**
https://github.com/Antiatics-Co/Lost-Islands-2D.git  Use EditorCollisionIssue Branch

**Screenshots (OPTIONAL):**
![image](https://github.com/user-attachments/assets/f489709b-95ca-41ed-a755-60038995a5d6)

**Additional context (OPTIONAL):
the Project has been through many versions of defold, I don't know if that would affect it or not

**Comment by britzl** [comment]
@OneWhoBuilds why was this closed? I believe it is still a problem that we plan to look into.

**Comment by OneWhoBuilds** [comment]
I figured that the case project would be of no use as the project was private, it has since been publicized.

Also the issue may be caused by the small tiles that made up the tilesource file. it had dimensions of 3 x 1 pixels.
Or the fact that the tilemap was imported from Tiled.

I made a new tilesource file with the dimensions 64 x 16, and replaced both the tilemap and tilesource files using the new image, and it seems to have fixed collision in the project. The lines in the editor are showing up and the collisions are showing up in physics debug mode
